src/SFA.DAS.FAT.Web/Infrastructure/CacheStorageValues.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.FAT.Web.Infrastructure;
+
+public static class CacheStorageValues
+{
+    public static class Providers
+    {
+        public const string Key = "RegisteredProviders";
+        public const int HoursToCache = 1;
+    }
```

Comment:
I like the idea, but since we are going to cache various other things and they will all need a key and time span, I would create a class with these two properties and then have static properties in here.
```
public record CacheInfo(string Key, TimeSpan CacheDuration);

public static class CacheKeys
{
  public const string RegisteredProviders = nameof(RegisteredProviders);
}

public static class CacheSetting
{  
  public static CacheInfo Providers => new(CacheKeys.RegisteredProviders, Timespan.FromHours(1));
}
```

Reply:
ok